ecoscape_connectivity/repopulation.py

```diff
@@ -31,7 +31,11 @@ def __init__(self, habitat, terrain, num_spreads=100, spread_size=1, min_transmi
         self.habitat = habitat
         self.goodness = torch.nn.Parameter(torch.max(habitat, terrain), requires_grad=True)
         self.h, self.w = habitat.shape
+        if callable(num_spreads):
+            num_spreads = num_spreads()
         self.num_spreads = num_spreads
+        if callable(spread_size):
+            spread_size = spread_size()
         self.spread_size = spread_size
         # Defines spread operator.
```

Comment:
It would be clearer to write: 
```
self.spread_size = spread_size() if callable(spread_size) else spread_size
```
Would you mind changing it in that way?